handbook/engineering/hiring/software-engineer-code-insights.md

```diff
@@ -0,0 +1,114 @@
+
+The code insights team is looking for an experienced, passionate engineer to build with us an entirely new product area of Sourcegraph: Code insights.
+Code insights is the first feature in Sourcegraph that provides **high-level** information about code, building atop all Sourcegraph's other features and even external data sources.
+
+We're building an [autonomous, international, full-stack team](https://about.sourcegraph.com/handbook/engineering/code-insights).
+
+The code insights team is building at the forefront of developer knowledge and code analysis using lean, user-focused methods – and everything we've built so far makes both our customers and our coworkers react with delight and surprise.
+
+## Why this job is exciting
+
+- You will be one of the **founding engineers** for this team. You will have a large, immediate impact on the shape and direction of code insights.
+- Despite its early stage, code insights is a heavily-validated feature: multiple Fortune-1000 companies have specifically requested early access to use our working prototypes.
+- Ahead lay extremely interesting challenges and opportunities for growth in making code insights scalable to hundreds of thousands of repositories, while also making it more powerful with filter capabilities, actionable insights, custom data sources – and much more we haven't even thought of yet.
+- In addition to serving all of Sourcegraph's existing developer users, code insights is the first feature we're building to serve engineering and product leaders at the director-or-above-level.
+- Sourcegraph's users all have technical backgrounds, meaning we can focus on the innovation and power of code insights rather than get caught up in product education.
+- Code insights will shape how all the top engineering-first companies build, organize, and evaluate themselves.
+
+## About you
+
+- You have practice at creating high quality software balanced with a pragmatic understanding of how to make appropriate tradeoffs (e.g. cut scope) to ship quickly and iterate when necessary.
+- You communicate clearly and empathetically, especially in writing and documentation.
+- You share our [values](../../../company/values.md), and work in accordance with those values.
+
+## Your responsibilities
+
+- Work closely with a Product Manager and Product Designer to implement a high-quality UI for code insights using React, SCSS, and TypeScript.
+- Actively shape the direction and participate in the planning of the features we're going to build, e.g. by writing and reviewing [RFCs](https://about.sourcegraph.com/handbook/communication/rfcs).
+- Build accessible UIs that other engineers and engineering leaders love using.
+- Collaborate with backend engineers to define the ideal APIs needed to deliver on the desired user experience.
+- Collaborate with other product teams like search, code intelligence and campaigns to figure out integration points with other Sourcegraph features.
+
+## Qualifications
```

Comment:
I'm not sure anymore, it probably just started to look like too many headings and like the bullet-point "profile" we are looking for was scattered across multiple headings instead of bundled. I had also looked at the [campaigns job description](https://about.sourcegraph.com/handbook/engineering/hiring/software-engineer-campaigns-full-stack-front-end#what-we-need-from-you), which doesn't have a "Qualifications"/"Nice to haves" split but just a single list, but I also remembered your comment about these sections being important for recruiters.

Reply:
(let me know if you want me to change this)